### PR TITLE
Fix big copy

### PR DESCRIPTION
**Pull request by ppolewicz**

### CODE REVIEW

File: test/unit/bucket/test_bucket.py [hunk 1]
```diff
@@ -218,6 +218,24 @@ def _make_data(self, approximate_length):
             fragments.append(fragment)
         return b''.join(fragments)
 
+    def _check_file_contents(self, file_name, expected_contents):
+        contents = self._download_file(file_name)
+        print(contents)
```
Comment (mzukowski-reef) [inline test/unit/bucket/test_bucket.py:223]: debug leftover?

File: b2sdk/transfer/emerge/emerger.py [hunk 2]
```diff
@@ -65,9 +67,17 @@ def emerge(
         :param dict,None file_info: a file info to store with the file or ``None`` to not store anything
         :param b2sdk.v2.AbstractProgressListener progress_listener: a progress listener object to use
 
+        :param int min_part_size: lower limit of part size for the transfer planner, in bytes
+        :param int max_part_size: upper limit of part size for the transfer planner, in bytes
         """
-        planner = self.get_emerge_planner(recommended_upload_part_size=recommended_upload_part_size)
-        emerge_plan = planner.get_emerge_plan(write_intents)
+        # WARNING: time spent trying to extract common parts of emerge() and emerge_stream()
+        # into a separate method: 20min. You can try it too, but please increment the timer honestly.
```
Comment (mzukowski-reef) [inline b2sdk/transfer/emerge/emerger.py:74]: add:
problematic lines are highlighted by  `# <--`
:)

**Review by mzukowski-reef — APPROVED** [review]

File: test/unit/bucket/test_bucket.py (hunk 1 above)
Comment (ppolewicz) [reply test/unit/bucket/test_bucket.py:223]: 😱

File: b2sdk/transfer/emerge/emerger.py (hunk 2 above)
Comment (ppolewicz) [reply b2sdk/transfer/emerge/emerger.py:74]: 👍

**Pull request merged**